contrib/datadog/tracing/interceptor.go

```diff
@@ -143,6 +143,18 @@ func (t *tracerImpl) ContextWithSpan(ctx context.Context, span interceptor.Trace
 	return tracer.ContextWithSpan(ctx, span.(*tracerSpan).Span)
 }
 
+// SpanFromWorkflowContext extracts the DataDog Span object from the workflow context.
+func SpanFromWorkflowContext(ctx workflow.Context) (ddtrace.Span, bool) {
```

Comment:
Not sure the bool is needed. We can just use `nil` to represent not present.

Reply:
Agreed - I've updated it now.

Reply:
Actually, I agree with @Quinn-With-Two-Ns below concerning noop span